rosdep/python.yaml

```diff
@@ -7351,6 +7351,26 @@ python3-openhsi-pip:
   ubuntu:
     pip:
       packages: [openhsi]
+python3-orjson:
+  arch: [python-orjson]
+  debian:
+    pip:
+      packages: [orjson]
+  fedora: [python3-orjson]
+  gentoo: [dev-python/orjson]
+  nixos: ['python%{python3_pkgversion}Packages.orjson']
+  openembedded:
+    pip:
+      packages: [orjson]
+  opensuse:
+    pip:
+      packages: [orjson]
+  rhel:
+    pip:
+      packages: [orjson]
+  ubuntu:
+    pip:
+      packages: [orjson]
```

Comment:
Normally native packages for Ubuntu and Debian are required for a non-pip key. I'm going to bring this one up at the next ROS meeting to see what others think.

Reply:
from the ROS meeting:

Please and a comment with a link to the [Debian intent-to-package bug](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1002996) for this package so that we know to replace the pip entry with that package when it becomes available.

Reply:
It is added